### ISSUE TITLE
Añadir link a youtube

### ISSUE THREAD

**Issue by franciscop** [makers-upv/web#3]
Posibilidad de añadir "vídeo de youtube" en vez de foto o flickr en el artículo. Poder combinarlo con fotos.

**Comment by juanvb** [comment]
https://cloud.githubusercontent.com/assets/11869115/15456112/f9a3a06c-2066-11e6-8c0c-5ff27a4421ec.png

**Comment by juanvb** [comment]
![](https://cloud.githubusercontent.com/assets/11869115/15456112/f9a3a06c-2066-11e6-8c0c-5ff27a4421ec.png)

**Comment by franciscop** [comment]
Espera, no habia visto la imagen. Lo que se puede cambiar es la imagen actual por un video, lo de meter un video intercalado en el texto depende de Google Docs y no de nosotros S:

Seria aceptable? Poner un video en lugar de la imagen actual y no como se indica en el mockup que has hecho.

**Comment by franciscop** [comment]
Bueno en verdad se podrian parsear todos los links, ver si uno es a youtube y sustituirlo por un iframe. Pero pasa de ser algo "facil" a ser algo "muy dificil y no se si es posible".

**Comment by juanvb** [comment]
Sí, un primer step es lo del vídeo en vez de la imagen, eso sería genial
paco! :)
Una vez implementado eso se puede pasar a una segunda fase que es lograr
que haya dos tipos de archivos multimedia en un mismo artículo.

*Juan Vicén BalaguerCofounder of Makers UPV http://www.makersupv.com/ *

2016-05-26 13:33 GMT+02:00 Francisco Presencia notifications@github.com:
